File: swift/larger.swift
import Foundation

print("Enter a num1: ", terminator: "")
if let input1 = readLine(), let num1 = Int(input1) {
    
    print("Enter a num2: ", terminator: "")
    if let input2 = readLine(), let num2 = Int(input2) {
        
        if num1 > num2 {
            print("num1 is larger: \(num1)")
        } 
         else {
            print("num2 is larger: \(num2)")
        }
     }
    }

File: java/larger.java
import java.util.Scanner;

class Scratch {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number1:");
        int num1 = scanner.nextInt();
        System.out.println("Enter the number2:");
        int num2 = scanner.nextInt();


        if (num1>num2){
            System.out.println("num1 is greater");
        }
        else{
            System.out.println("num2 is greater");
        }


        scanner.close();
    }
}